repository/t.json

```diff
@@ -3246,6 +3246,17 @@
 				}
 			]
 		},
+		{
+			"name": "Trusted Coding",
+			"details": "https://bitbucket.org/madnext/trustedcodingsublime",
```

Comment:
You might want to add some labels for categorization of your package.

Reply:
I just added some labels. Thanks! :)